js/jquery.mapael.js

```diff
@@ -237,13 +237,7 @@
 					, elemOptions = {};
 				
 				// Reset hidden map elements (when user click on legend elements)
-				$.each(legends, function(index, el) {
-					el.forEach && el.forEach(function(el) {
-						if(typeof el.hidden != "undefined" && el.hidden == true) {
-							$(el.node).trigger("click");
-						}
-					});
-				});
+				$self.trigger("hideAllElems", true);
```

Comment:
Wow, I miss an issue... This should be set to false of course... 
```
$self.trigger("hideAllElems", false);
```

My bad...